roles/kubernetes-apps/ansible/tasks/coredns.yml

```diff
@@ -1,21 +1,34 @@
 ---
-- name: Kubernetes Apps | Lay Down CoreDNS Template
-  action: "{{ item.module }}"
-  args:
-    src: "{{ item.file }}{% if item.module == 'template' %}.j2{% endif %}"
+- name: Kubernetes Apps | Lay Down CoreDNS templates
+  template:
+    src: "{{ item.file }}.j2"
     dest: "{{ kube_config_dir }}/{{ item.file }}"
-  with_items:
+  loop:
     - { name: coredns, module: template, file: coredns-config.yml, type: configmap }
```

Comment:
I don't really see the point in keeping all those, this was useful when the task was using kube module, but in the loop we are only using the `file` key.. I think we could go with
```
loop:
  - coredns-config.yml
  - coredns-deployment.yml
[...]
```` 
And only use `{{ item }}`